INSTRUCTION.md

```diff
@@ -0,0 +1,11 @@
+how to run MySQL container with a volume attached
+docker run -d -p 8080:8080 --name mysql-local -v mysql-volume mysql-local:1.0.0
+
+how to run an App container which connected to a MySQL db container to a MySQL db container.
```

Comment:
The section title is repeated and contains a typo. It should be corrected to 'how to run an App container connected to a MySQL db container.'